client/js/widgets/button.js

```diff
@@ -109,6 +109,29 @@ export class Button extends Widget {
                 w.click();
       }
 
+      if(a.func == 'CLONE'){
+        setDefaults(a, { source: 'DEFAULT', count: 1, xOffset: 0, yOffset: 0, collection: 'DEFAULT' });
+        if(a.properties.applyVariables) {
+          this.applyVariables(a.properties, variables, problems);
+          delete a.properties["applyVariables"];
+        };
+        if(isValidCollection(a.source)) {
+          var c=[];
+          for(const w of collections[a.source]) {
+            const clone = Object.assign(JSON.parse(JSON.stringify(w.state)), a.properties);
+            clone.x = clone.x + a.xOffset;
+            clone.y = clone.y + a.yOffset;
+            clone.clonedFrom = w.p('id');
+            for(let i=0; i<a.count; ++i) {
+              clone.id = null;
```

Comment:
I would do `delete clone.id` here but it probably doesn't make a difference.

Reply:
The test later is `if (!widget.id)`, so it should behave the same.